inc/wizard.class.php

```diff
@@ -165,6 +168,56 @@ public static function header($title) {
          }
       }
 
+      $link = new Link;
+      $ar_links = $link->find("name LIKE '". $_SESSION['plugin_formcretor']['external_links_prefix'] ."%'");
+      foreach ($ar_links as $id => $a_link) {
+         /*
+          * The link name is filtered and trimmed to remove the 'Helpdesk' string and extra whitespaces:
+          * Helpdesk 1 - the first link
+          * will display: the first link.
+          */
+         $label = trim(str_replace($_SESSION['plugin_formcretor']['external_links_prefix'], '', $a_link['name']));
```

Comment:
I suggest you move this line in a public method (in this class or in PluginFormcreatorCommon), to make easy to unit test it. Especially because you (or someone else) might improve it.

Reply:
Created a method cleanPattern in the PluginFormcreatorCommon class